app/javascript/src/component_dialog_form.js

```diff
@@ -75,21 +76,21 @@ class FormDialog {
     ];
 
     // Sort out easy state reporting.
-    this.#state = STATE_CLOSED;
+    this.#state = FormDialog.STATE_CLOSED;
 
     $node.on("dialogclose", function( event, ui ) {
-      dialog.$container.attr(STATE_ATTR, STATE_CLOSED);
+      dialog.$container.attr(FormDialog.STATE_ATTR, FormDialog.STATE_CLOSED);
     });
 
     $node.on("dialogopen", function( event, ui ) {
-      dialog.$container.attr(STATE_ATTR, STATE_OPEN);
+      dialog.$container.attr(FormDialog.STATE_ATTR, FormDialog.STATE_OPEN);
     });
```

Comment:
Should this not set `#state`?

Reply:
#state is left over from my original effort/plan but is no longer used. I have now removed.